src/main/java/frc/robot/subsystems/ArmPivot.java

```diff
@@ -1,14 +1,49 @@
 package frc.robot.subsystems;
 
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.configs.TalonFXConfigurator;
 import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.Hardware;
+import java.util.function.DoubleSupplier;
 
 public class ArmPivot extends SubsystemBase {
+  // Presets
+  public static int PRESET_L1 = 0;
+  public static int PRESET_L2_L3 = 35;
+  public static int PRESET_L4;
+  public static int HARDSTOP_HIGH = 90;
+  public static int HARDSTOP_LOW = 0;
+  public static double placeholderCoralWeight;
+  public static double placeholderAlgaeWeight;
```

Comment:
Since these should be constants, change to using all caps with underscores like above. And specify the weight units (ideally `_KG` to align with the default units WPILib uses)